ksql-engine/src/test/java/io/confluent/ksql/analyzer/AnalyzerTest.java

```diff
@@ -153,67 +156,46 @@ public void testSimpleQueryAnalysis() {
+            + "FROM test1 t1 LEFT JOIN test2 t2 "
+            + "ON t1.col1 = t2.col1;", jsonMetaStore);
 
-    Assert.assertNotNull("SELECT is null", analysis.getSelectExpressions());
-    Assert.assertNotNull("SELECT aliacs is null", analysis.getSelectExpressionAlias());
-    Assert.assertTrue("JOIN left hand side was not analyzed correctly.",
-                      analysis.getJoin().getLeftAlias().equalsIgnoreCase("t1"));
-    Assert.assertTrue("JOIN right hand side was not analyzed correctly.",
-                      analysis.getJoin().getRightAlias().equalsIgnoreCase("t2"));
+    // Then:
+    assertThat(analysis.getFromDataSources(), hasSize(2));
+    assertThat(analysis.getFromDataSources().get(0).getAlias(), is("T1"));
+    assertThat(analysis.getFromDataSources().get(1).getAlias(), is("T2"));
 
-    Assert.assertEquals(analysis.getSelectExpressions().size(),
-        analysis.getSelectExpressionAlias().size());
+    assertThat(analysis.getSelectExpressions().size(),
+        is(analysis.getSelectExpressionAlias().size()));
 
-    assertThat(analysis.getJoin().getLeftJoinFieldName(), is("T1.COL1"));
-    assertThat(analysis.getJoin().getRightJoinFieldName(), is("T2.COL1"));
+    assertThat(analysis.getJoin(), is(not(Optional.empty())));
+    assertThat(analysis.getJoin().get().getLeftJoinField(), is("T1.COL1"));
+    assertThat(analysis.getJoin().get().getRightJoinField(), is("T2.COL1"));
 
-    final String
-        select1 =
-        SqlFormatter.formatSql(analysis.getSelectExpressions().get(0))
-            .replace("\n", " ");
-    Assert.assertTrue(select1.equalsIgnoreCase("T1.COL1"));
-    final String
-        select2 =
-        SqlFormatter.formatSql(analysis.getSelectExpressions().get(1))
-            .replace("\n", " ");
-    Assert.assertTrue(select2.equalsIgnoreCase("T2.COL1"));
-    final String
-        select3 =
-        SqlFormatter.formatSql(analysis.getSelectExpressions().get(2))
-            .replace("\n", " ");
-    final String
-        select4 =
-        SqlFormatter.formatSql(analysis.getSelectExpressions().get(3))
-            .replace("\n", " ");
-    Assert.assertTrue(select3.equalsIgnoreCase("T2.COL4"));
-    Assert.assertTrue(select4.equalsIgnoreCase("T1.COL5"));
-
-    Assert.assertTrue(analysis.getSelectExpressionAlias().get(0).equalsIgnoreCase("T1_COL1"));
-    Assert.assertTrue(analysis.getSelectExpressionAlias().get(1).equalsIgnoreCase("T2_COL1"));
-    Assert.assertTrue(analysis.getSelectExpressionAlias().get(2).equalsIgnoreCase("T2_COL4"));
-    Assert.assertTrue(analysis.getSelectExpressionAlias().get(3).equalsIgnoreCase("COL5"));
-    Assert.assertTrue(analysis.getSelectExpressionAlias().get(4).equalsIgnoreCase("T2_COL2"));
+    final List<String> selects = analysis.getSelectExpressions().stream()
+        .map(SqlFormatter::formatSql)
+        .collect(Collectors.toList());
+
+    assertThat(selects, contains("T1.COL1", "T2.COL1", "T2.COL4", "T1.COL5", "T2.COL2"));
```

Comment:
The order of these is important, right? The new version of this test doesn't appear to check order.

Reply:
`contains` checks order.